client/boot/common.js

```diff
@@ -402,10 +404,22 @@ const setupMiddlewares = ( currentUser, reduxStore, reactQueryClient ) => {
 	}
 };
 
+let wpcomRootNode;
+export const getRootNode = () => {
```

Comment:
A better name would be `getReactRoot`. The `getRootNode` name suggests that the return value is a DOM node, but it's a React root object.